packages/frontend-shared/src/locales/en-US.json

```diff
@@ -121,7 +120,15 @@
     "searchPlaceholder": "Search Specs",
     "componentSpecsHeader": "Component specs",
     "e2eSpecsHeader": "E2E specs",
-    "lastUpdatedHeader": "Last updated",
+    "lastUpdated": {
+      "header": "Last updated",
+      "tooltip": {
```

Comment:
We don't need this tooltip text in the i18n just yet, might as well remove it until the PR with the header updates comes along? Not a blocking request though.